### ISSUE TITLE
Arm deployment for Azure Data Factory failed without error message

### ISSUE THREAD

**Issue by andiasmara** [Azure/arm-deploy#33]
We try to implement CI/CD for Azure Data Factory (https://docs.microsoft.com/en-us/azure/data-factory/continuous-integration-deployment) using Github Action. In Actions, we use azure/arm-deploy@v1 for ARM deployment.
It was working fine last month, and then a few weeks ago, it started to failed without error message.

<img width="630" alt="Screenshot 2021-02-09 at 16 26 23" src="https://user-images.githubusercontent.com/37690064/107943394-edc2c000-6f8c-11eb-8d10-c36736559311.png">

According to Microsoft Support, there is nothing wrong with the ARM template itself and they asked us to open an issue here.

**Comment by kanika1894** [comment]
@andiasmara
Can you try running the action in debug mode to get more helpful logs?
You can refer https://docs.github.com/en/free-pro-team@latest/actions/managing-workflow-runs/enabling-debug-logging#enabling-runner-diagnostic-logging to enable the debug mode.
It will be great if you can share the failed run.

**Comment by andiasmara** [comment]
@kanika1894 
Should I enable "runner diagnostic logging" only or "step debug logging" as well? 
and how can I share the log with you? there are some quite sensitive data in the log.

**Comment by kanika1894** [comment]
To enable step debug logging, you must set the following secret in the repository that contains the workflow: ACTIONS_STEP_DEBUG to true.
After setting the secret, more debug events are shown in the step logs. That can help you identify the issue.

**Comment by vineetmimrot** [comment]
@andiasmara Can you please provide logs as mentioned by @kanika1894 ?
Since you mentioned it contains sensitive data, can you send it to 
![image](https://user-images.githubusercontent.com/4174297/110466233-83b8b900-80fb-11eb-91cc-644a2a89b334.png)

**Comment by vineetmimrot** [comment]
@andiasmara Any update?

**Comment by kanika1894** [comment]
@andiasmara waiting for your inputs.

**Comment by andiasmara** [comment]
@vineetmimrot and @kanika1894. Have you received the log files?

**Comment by vineetmimrot** [comment]
@andiasmara Yes we have received the logs and @tejasd1990 is working on this issue.

**Comment by marvinbuss** [comment]
We are running into the same issue with another ARM template. We tried to deploy Private DNS Zones and came across the same issue. Do you have enough logs or do you need more data @tejasd1990 & @vineetmimrot ?

**Comment by tejasd1990** [comment]
Hi all,
we have identified the root cause of this issue, will inform once we have an update regarding the fix checkin and rollout.

**Comment by marvinbuss** [comment]
Would be great if you could rollout the bugfix until tomorrow afternoon CET, since we are relying on this Action for our deployments.

**Comment by marvinbuss** [comment]
@tejasd1990 Can we expect a fix later today?
Thanks for your help!

**Comment by tejasd1990** [comment]
Hi @marvinbuss , we are still figuring out the fix, i will update once it is ready.

**Comment by tejasd1990** [comment]
Hi all,
the fix has been deployed. Let us know if you are still facing the issue

**Comment by jontreynes** [comment]
Possibly this will show more info
#40

**Comment by t-dedah** [comment]
Closing this issue due to inactivity. Please reopen if you are still facing the same issue. @andiasmara
